tidoop-mr-lib-api/src/mysql_driver.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-tidoop (FI-WARE project).
+ *
+ * fiware-tidoop is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-tidoop. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * MySQL driver.
+ *
+ * Author: frb
+ */
+
+// module dependencies
+var mysql = require('mysql');
+var host = require('../conf/tidoop-mr-lib-api.json').mysql.host;
+var port = require('../conf/tidoop-mr-lib-api.json').mysql.port;
+var user = require('../conf/tidoop-mr-lib-api.json').mysql.user;
+var password = require('../conf/tidoop-mr-lib-api.json').mysql.password;
+var database = require('../conf/tidoop-mr-lib-api.json').mysql.database;
+
+var connection = mysql.createConnection({
+    host: host,
+    port: port,
+    user: user,
+    password: password,
+    database: database
+});
+
+module.exports = {
+    connect: function() {
```

Comment:
Since this module is really a library, maybe you would prefer to include a callback in this `connect` function to inform the calling part about the result of the call. I mean, something such as:
```
connect: function() {
  connection.connect(function (error) {
    callback(error);
});
```